Makefile

```diff
@@ -48,6 +48,13 @@ PROJECT_SLUG := github.com/vmware-tanzu/vm-operator
 # ROOT_DIR_IN_GOPATH is non-empty if ROOT_DIR is in the GOPATH.
 ROOT_DIR_IN_GOPATH := $(findstring $(GOPATH)/src/$(PROJECT_SLUG),$(ROOT_DIR))
 
+# R_WILDCARD recursively searches for a file that matches $1.
+R_WILDCARD=$(wildcard $(1)$(2)) $(foreach DIR,$(wildcard $(1)*),$(call R_WILDCARD,$(DIR)/,$(2)))
```

Comment:
Also, I think `$(foreach DIR,$(wildcard $(1)*)` also lists *files* in $1, so you should change it to `$(foreach DIR,$(wildcard $(1)*/)`?

Reply:
It doesn't seem to work because `$(call R_WILDCARD,$(DIR)/,$(2)))` would add an extra "/" at the end, making the path invalid in recursive calls.